deploy.sh

```diff
@@ -6,8 +6,17 @@ git config user.name "Travis CI"
 git config user.email "rennierab@gmail.com"
 
 git checkout master
+git stash
+git stash apply
+git add dist/anything.js
+git add dist/anything.min.js
+git commit -m "Update dist [ci skip]"
+
+git checkout gh-pages
```

Comment:
I don't know how the environment is setup inside Travis, so I don't know if this is going to work.